configure.ac

```diff
@@ -88,7 +88,7 @@ AS_IF([test "x${enable_flags_setting}" = "xyes" && test "x${enable_debug}" = "xn
 	CFLAGS=$( echo ${CFLAGS} | $SED -e 's/^ *//' -e 's/ *$//' )
 
 	AC_LANG([C++])
-	AX_APPEND_COMPILE_FLAGS([${cflags_test}], [CXXFLAGS])
+	AX_APPEND_COMPILE_FLAGS([${cflags_test} -std=gnu++11], [CXXFLAGS])
```

Comment:
Wrong macro, please use https://www.gnu.org/software/autoconf-archive/ax_cxx_compile_stdcxx.html#ax_cxx_compile_stdcxx
`AX_CXX_COMPILE_STDCXX([11], [noext], [mandatory])`

Reply:
Thank you for reminding me that one I've completely forgotten it...